timely_beliefs/tests/conftest.py

```diff
@@ -12,12 +13,16 @@
 )
 from timely_beliefs.tests import engine, session
 
+collect_ignore = []
 if sys.version_info[0] == 3 and sys.version_info[1] == 6:
     # Ignore these tests for python==3.6
-    collect_ignore = ["test_ignore_36.py", "test_plotting.py", "test_ridgeline.py"]
-if sys.version_info[0] == 3 and sys.version_info[1] == 7:
-    # Ignore these tests for python==3.7
-    collect_ignore = ["test_plotting.py", "test_ridgeline.py"]
+    collect_ignore += ["test_ignore_36.py"]
+if importlib.util.find_spec("altair") is None:
+    # Ignore these tests if altair is not installed
+    collect_ignore += ["test_viz__plotting.py", "test_viz__ridgeline.py"]
+if importlib.util.find_spec("sktime") is None:
+    # Ignore these tests if sktime is not installed
```

Comment:
I think I see the reasoning behind this - when someone isn't using one of our `make test[X]` functions, they'd fail these tests if they have not installed everything.

Is it a good idea to print out that this is what is happening?